test/integration/addons_test.go

```diff
@@ -796,6 +797,19 @@ func validateHeadlampAddon(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+func validateInspektorGadgetAddon(ctx context.Context, t *testing.T, profile string) {
```

Comment:
Could you add a comment above in the format `// validateInspektorGadgetAddon <explanation>` and then run `make generate-docs` to update the list of tests on the website.